modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -1691,6 +1711,233 @@ void testAddColumnWithExistingName() {
         assertThat(manager.execute(addColumnParams(columnParams("ID", INT32))), willThrowFast(CatalogValidationException.class));
     }
 
+    @Test
+    public void testCreateSystemView() {
```

Comment:
this class is over 2k lines... let's move all new tests (which are related to introduced system view command) to the separate class

Reply:
Done